### ISSUE TITLE
Make it possible to build project on node.js v16

### ISSUE THREAD

**Issue by vladimiry** [ProtonMail/proton-mail#74]
Currently `npm ci` step gets failed on node.js v16 during rebuilding `sharp` native module.

Upgrading `sharp` version in `proton-pack#favicons-webpack-plugin#favicons` and `proton-pack#sharp` chains is likely going to help, see https://github.com/lovell/sharp/issues/2611.

**Comment by dhoko** [comment]
On our side, we only validate things for the latest LTS (v14).
Compat with the new LTS will be done before the v16 becomes the LTS. It should be done next month maybe I think :thinking: 

(Depends if the support breaks  something for v14 or we can have it for both, LTS is more important than v16)

**Comment by vladimiry** [comment]
Yep, I understand the LTS matter. Just wanted to highlight that updating `sharp` version will fix the issue when running on node.js v16.
